src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const remoweRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+addRow.addEventListener('click', () => {
+  if(table.rows.length === 10) {
+    return;
+  }
+  const rowToClone = table.rows[0];
+  const newRow = rowToClone.cloneNode(true);
+  table.appendChild(newRow);
+});
+
+remoweRow.addEventListener('click', () => {
```

Comment:
Use the correct variable name `removeRow` instead of `remoweRow` when adding the event listener. 🚀